docs/i18n.rst

```diff
@@ -119,10 +104,8 @@ Python code, HTML templates, or desktop icon labels, the translation
 files should also be updated by running ``make translate`` in the root
```

Comment:
Per below, we'll name this target more explicitly going forward (and elsewhere in this document):

```suggestion
files should also be updated by running ``make extract-strings`` in the root
```

Yet to be determined: whether this is enforced at commit-time or only in CI.